nUpdate.ProvideTAP/Updating/UpdateConfiguration.cs

```diff
@@ -23,9 +23,10 @@ public partial class UpdateConfiguration
         ///     The optional <see cref="CancellationTokenSource" /> to use for canceling the
         ///     operation.
         /// </param>
+        /// <param name="timeout">The timeout of dowloading configuration file. In milliseconds. Default 10000.</param>
```

Comment:
Code should be fine. Please do only edit the documentation to make it more unique. Additionally, there is a small typo in"dowloading". Something like "The timeout for the download request" should be it.